bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -882,6 +907,112 @@ public ClientConfiguration setBookieErrorThresholdPerInterval(long thresholdPerI
     }
 
     /**
+    * Whether the client uses SSL or not. By default the client does not use SSL.
+    * @return whether the client uses SSL or not
+    */
+    public boolean getUseSSL() {
+        return getBoolean(USE_SSL, false);
+    }
+
+    /**
+     * Specify whether the client should use SSL or not. BookKeeper uses STARTTLS,
+     * so both SSL communication and non-SSL communication runs on the same port.
+     * If the client is configured to use SSL, and no SSL enabled bookies are available
+     * the client will not be able to connect to the cluster.
+     *
+     * @param useSSL whether to use SSL or not
+     * @return client configuration
+     */
+    public ClientConfiguration setUseSSL(boolean useSSL) {
+        setProperty(USE_SSL, useSSL);
+        return this;
+    }
+
+    /**
+    * Whether the client validates SSL certificates. It defaults to false
+    * @return whether the client will verify certificates or not.
+    */
+    public boolean getVerifySSLCertificates() {
+        return getBoolean(VERIFY_SSL_CERTIFICATES, false);
+    }
+
+    /**
+    * Specify whether the client validates SSL certificates
+    * @return whether the client will verify certificates or not.
+    */
+    public ClientConfiguration setVerifySSLCertificates(boolean value) {
+        setProperty(VERIFY_SSL_CERTIFICATES, value);
+        return this;
+    }
+
+
+    /**
+      * Specify whether the client will send an SSL certificate on TLS-handshake
+      * @param Whether to send a certificate or not
+      * @return the server configuration
+      */
+     public ClientConfiguration setClientSSLAuthentication(boolean enabled) {
+         setProperty(CLIENT_SSL_AUTHENTICATION, enabled);
+         return this;
+     }
+
+     /**
+      * Whether the client will send an SSL certificate on TLS-handshake
+      * @see #setClientSSLAuthentication(boolean)
+      * @return whether SSL is enabled on the bookie or not.
+      */
+     public boolean isClientSSLAuthentication() {
```

Comment:
Reword function name or keep it consistent with the above convention of "set" or "get"